src/shared/components/views/VisualFeaturesView.tsx

```diff
@@ -5,13 +5,16 @@ import { Link, useParams } from 'react-router-dom';
 import NightingaleZoomTool, {
   iconSize,
 } from '../../../uniprotkb/components/protein-data-views/NightingaleZoomTool';
-import { TabLocation } from '../../../uniprotkb/components/entry/Entry';
 
 import useCustomElement from '../../hooks/useCustomElement';
+
 import { getEntryPath } from '../../../app/config/urls';
+import { gtagFn } from '../../utils/logging';
+
+import { TabLocation } from '../../../uniprotkb/components/entry/Entry';
 import { Namespace } from '../../types/namespaces';
 
-import './styles/visual-features-view.scss';
+import styles from './styles/visual-features-view.module.scss';
```

Comment:
We tend to use CSS modules wherever we can (especially in new styles or when touching old ones) to avoid involuntary global styles / leaks to other components